lms/djangoapps/instructor/enrollment.py

```diff
@@ -287,6 +295,39 @@ def _reset_module_attempts(studentmodule):
     studentmodule.save()
 
 
+def _fire_score_changed_for_block(course_id, student, block, module_state_key):
+    """
+    Fires a SCORE_CHANGED event for the given module. The earned points are
+    always zero. We must retrieve the possible points from the XModule, as
+    noted below.
+    """
+    cache = FieldDataCache.cache_for_descriptor_descendents(
+        course_id=course_id,
+        user=student,
+        descriptor=block,
+        depth=0
+    )
+    # For implementation reasons, we need to pull the max_score from the XModule, even though the data is not user-
+    # specific.  Here we bind the data to the current user.
+    request = crum.get_current_request()
```

Comment:
Just to be clear: the reason I didn't change the function that calls this to take a whole request object instead of a requesting user is this code:

https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor/services.py#L35

This is a library that edx-proctoring imports so that it can call edx-platform functionality, specifically deleting student state. It did not seem worth it to me to go through creating a new release of the platform and updating edx-proctoring right now. I'd consider this tech debt.